### DISCUSSION TITLE
How to force placement value despite not having enough space?

### DISCUSSION THREAD

**Discussion by septem1997** [ariakit/ariakit#3657] [Questions] [answered]
here is a demo: https://codesandbox.io/p/sandbox/flamboyant-ride-24j326?file=%2Findex.js%3A10%2C22

I set placement as bottom, but it only works when there is enough space. How can I force it to use the value I set?

<img width="1511" alt="image" src="https://github.com/ariakit/ariakit/assets/13849245/43ded3c3-0b3f-41ee-b742-6d1b4e1889e4">

**Comment by diegohaz** [comment, marked as answer]
Set [`flip`](https://ariakit.org/reference/combobox-popover#flip) to `false` on `ComboboxPopover`:

```jsx
<ComboboxPopover flip={false}>
```
